.vitepress/config.ts

```diff
@@ -49,7 +49,7 @@ const nav: ThemeConfig['nav'] = [
           { text: 'UI Components', link: 'https://ui-libs.vercel.app/' },
           {
             text: 'Certification',
-            link: 'https://certification.vuejs.org/?ref=vuejs-nav'
+            link: 'https://certificates.dev/vuejs/?friend=VUEJS&ref=vuejs-nav&utm_source=vuejs&utm_medium=link&utm_campaign=vuejs_nav'
```

Comment:
**suggestion**: I believe we can keep the link only with ref, without `utm_source` and all the other query params:

```suggestion
            link: 'https://certificates.dev/vuejs/?ref=vuejs-nav'
```